src/lando/utils/management/commands/tests.py

```diff
@@ -22,9 +22,17 @@ def add_arguments(self, parser):
             "paths", nargs="*", type=str, help="Files or directories to pass to pytest"
         )
 
+        parser.add_argument(
+            "-k", type=str, help="Only run tests which match the given substring."
+        )
+
     def handle(self, *args, **options):
         command = ["pytest"]
 
+        if options["k"]:
+            command.append("-k")
+            command.append(options["k"])
+
```

Comment:
This also is out of scope for this PR, and ideally would be moved to its own PR.

Reply:
#150